.github/DISCUSSION_TEMPLATE/q-a-installation.yml

```diff
@@ -0,0 +1,47 @@
+body:
+- type: input
+  attributes:
+    label: Operating System
+    description: "What OS are you using?"
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: RAM size
+    description: "What is the RAM size?"
+  validations:
+    required: true
+- type: textarea
+  id: steps
+  attributes:
+    label: Which step are you stuck in?
+    description: |
+        If you encountered this error while following a wiki page, provide a link to the page and specify which step failed.
+        Otherwise, list what steps caused the error.
+        These should be detailed enough for someone else to follow them.
+  validations:
+    required: false
```

Comment:
Shouldn't this be required? Even if they weren't following a wiki, we want them to tell us what causes the error

Reply:
Done